render/drm_dumb_allocator.c

```diff
@@ -0,0 +1,212 @@
+#define _POSIX_C_SOURCE 200809L
+#include "render/drm_dumb_allocator.h"
```

Comment:
Style: the order of these includes isn't consistent with the rest of wlroots. In general we do first the angle-bracket includes (sorted alphabetically) then the double-quote includes (sorted alphabetically).